### DISCUSSION TITLE
Do you have an official Nodejs client for this package?

### DISCUSSION THREAD

**Discussion by nabi-ebrahimi** [ScrapeGraphAI/Scrapegraph-ai#378] [General]
I already have NestJs system, i love this package, so i want to add this in my system, do you have any plan for creating a nodejs package? what is the best way to add this in my system. thanks

**Comment by VinciGit00** [comment]
You mean something like that? 

https://github.com/mishushakov/llm-scraper

**Reply by nabi-ebrahimi** [reply]
@VinciGit00 thanks for your reply. Yes exactly, do you have any plan to work on it?

**Reply by VinciGit00** [reply]
We are python based, if there is a way to embedd it we can work togheter
